### ISSUE TITLE
otel_ngx_module: New release for nginx 1.25.3

### ISSUE THREAD

**Issue by nise-wg2** [open-telemetry/opentelemetry-cpp-contrib#344]
After downloading the so from https://github.com/open-telemetry/opentelemetry-cpp-contrib/actions, it looks like If we use otel_ngx_module.so library with NGINX 1.25.3 version we are getting below error:
module "/opt/opentelemetry-cpp-contrib/otel_ngx_module.so" version 1024000 instead of 1025003

Can there be a proper release of otel_ngx_module that supports nginx 1.25.3 ? or an easy approach to build it for the newer version of Nginx?

**Comment by codedinsugar** [comment]
@nise-wg2 did you ever get an answer for this?  Also, how did you download the `otel_ngx_module.so` file?  All of the artifacts appear to be expired here:

https://github.com/open-telemetry/opentelemetry-cpp-contrib/actions/workflows/nginx.yml

**Comment by nise-wg2** [comment]
No, I ended up using github.com/open-telemetry/opentelemetry-cpp-contrib/releases/download/webserver
Which is noisy and not configurable in the same way.
